### PR TITLE
Remove leaking boost dependency

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

**Comment by tribal-tec** [comment]
hold this, I found more...

**Comment by tribal-tec** [comment]
Ok, ready now

File: eq/fabric/errorRegistry.h [hunk 1]
```diff
@@ -28,40 +27,43 @@ namespace fabric
...
+    /** Clear a given error code string. @version 1.0 */
+    EQFABRIC_API void eraseString( const uint32_t error );
 
-        /** Clear a given error code string. @version 1.0 */
-        EQFABRIC_API void eraseString( const uint32_t error );
+    EQFABRIC_API bool isEmpty() const; //!< @internal
 
-        EQFABRIC_API bool isEmpty() const; //!< @internal
+private:
+    ErrorRegistry( const ErrorRegistry& );
```
Comment (rdumusc) [inline eq/fabric/errorRegistry.h]: = delete here too for consitency

**Comment by rdumusc** [comment]
+1 otherwise.

**Pull request merged**